recipes/cad1/README.md

```diff
@@ -0,0 +1,91 @@
+# The 1st Cadenza Challenge
+Cadenza challenge code for the 1st challenge.
+
+For more information about the Cadenza Challenge please visit https://cadenzachallenge.com
+
+Cadenza tutorials are now available in https://cadenzachallenge.com.
+The tutorials introduce the Cadenza installation, how to interact with Cadenza metadata,
```

Comment:
It may be in the wings waiting to be deployed but the learning resources (see above comment) don't include this information. I think it might be useful to link to the specific pages for each of these, perhaps as a bullet list as suggested above.

Reply:
I can't find where these tutorials are nor where the code for the site (https://cadenzachallenge.org/) resides (none of the repos within this organisation jumped out as being the one).

It might be worth capturing the completion of these pages as an Issue either within Clarity or on the repository for the site if its its under version control/GitHub.